vite.config.ts

```diff
@@ -120,5 +120,6 @@ export default defineConfig(({ mode }) => ({
       { find: /^@\//, replacement: '/src/' },
       { find: 'goosefx-amm-sdk', replacement: '/gamma-sdk' }
     ]
-  }
+  },
+  assetsInclude: ['gamma-wasm/gamma_wasm_bg.wasm']
```

Comment:
have you ran `yarn build` + `yarn serve` to check this works. If so awesome stuff :D

Reply:
yeah those commands work